packages/code-editor/src/customTabs/Markdown/index.js

```diff
@@ -22,11 +22,14 @@ import './styles.scss'
 // import ForkButton from './ForkButton'
 
 export default class Markdown extends Component {
-  // state = {
-  //   avatar: ''
-  // }
```

Comment:
may need to confirm if it should be kept, if not, can delete the related code block